resources/assets/coffee/_classes/forum-cover.coffee

```diff
@@ -62,24 +62,23 @@ class @ForumCover
     @uploadButton.length > 0
 
 
-  toggleModal: (e) =>
-    e.preventDefault()
+  isModalOpen: (isModalOpen) =>
+    return false if !@hasCoverEditor()
 
-    if @header[0]._open
-      @closeModal()
-    else
-      @openModal()
+    if isModalOpen?
```

Comment:
It's misleading to allow something named `isXXX` to have values other than `true` and `false`.